ethstats/ethstats.go

```diff
@@ -981,11 +981,9 @@ func (s *Service) assembleBlockStats(block *types.Block) *blockStats {
 	// Gather the block infos from the local blockchain
 	var (
 		header   *types.Header
-		stateDB  *state.StateDB
-		vmRunner vm.EVMRunner
-		td       *big.Int
```

Comment:
Just wondering why you moved td? Seems like it doesn't change the functionality but more likely to cause a conflict in a future merge.

Reply:
The vmRunner is ours. Don't really see this possible conflict as an issue, haha

Reply:
Good call. I reverted the change to `td`. The others I don't think should introduce any conflict.

Reply:
Thanks, moving vm and stateDB I think will reduce conflicts, since now that var block perfectly matches upstream